index.html

```diff
@@ -1002,6 +1002,39 @@ <h2 class='informative'>Implementation Patterns</h2>
       </section>
     </section>
 
+    <section id="privacy">
+      <h2>Privacy Considerations</h2>
+      <p>
+        Fedora resources may contain personally identifiable information such as the identify of
+        the agents who create and modify resources. As such, this information could be analyzed
+        to generate profiles of the behavior of individuals or groups.
+      </p>
+      <p>
+        As such, Fedora implementations that produce or consume personally identifiable
+        information MUST clearly document the type and extent of personal information that
+        is collected and published along with the reasons for doing so.
```

Comment:
I'm not dead-set against it, but I don't think it has much value and I think it is _way_ outside the remit of this spec to be discussing these issues in any particular way.